### ISSUE TITLE
Support glTF KHR_materials_unlit

### ISSUE THREAD

**Issue by daisuke201** [sony/mapray-js#115]
## Background
Below is a summary of the changes.

- (A) Support extensionsRequired property of glTF content
- (B) Corresponds to extensionsUsed property of glTF content and common property of object
- (C) ModelMaterial class can be used without lighting

(A) will cause load to abort correctly when the content uses an extension that the glTF loader or glTF renderer cannot support.

(B) makes it possible to refer to the name, extensions, and extra properties of each glTF object. Only the extensions listed in the extensionsUsed property of the top-level object can be obtained from the extensions property.

Due to (C), materials that have "KHR_materials_unlit" in their extensions property will not be lit. The ModelMaterial class uses the shader for no lighting internally when the is_unlit option is given.

Create ModelMaterial instances for general and no lighting inside ModelContainer. Shader properties are set the same for both instances.

The shader source code uses shader / model. {Vert, frag} in both modes to generate the code to compile with the define macro.

For details of extensionsRequired, extensionsUsed, extensions properties, refer to [Specifying Extensions](https://github.com/KhronosGroup/glTF/tree/master/specification/2.0#specifying-extensions).

This material  might be used by photogrammetry model from drone etc.

**Comment by daisuke201** [comment]
The lower duck is a normal material and the upper duck is a material using KHR_materials_unlit

![75888380-a1ab6800-5e6e-11ea-82a6-05ae2382a97b](https://user-images.githubusercontent.com/9956091/79203299-cc2d0f80-7e75-11ea-9aaa-3c142c056b24.png)

**Comment by daisuke201** [comment]
Released this issue with 0.8.1

**Comment by daisuke201** [comment]
## How to use KHR_materials_unlit
1. Add "KHR_materials_unlit" to the top-level extensionsUsed property as follows:

```
"extensionsUsed": [
    "KHR_materials_unlit"
]
```

2. Add an empty object with the key "KHR_materials_unlit" to the extensions property of the non-lighting material.

```
"materials": [
    {
        "pbrMetallicRoughness": {
            "baseColorTexture": {
                "index": 0
            }
        },
        "extensions": {
            "KHR_materials_unlit": {}
        }
    }
]
```

## Sample file
[KhronosGroup glTF sample/UnlitTest ](https://github.com/KhronosGroup/glTF-Sample-Models/tree/master/2.0/UnlitTest)
